templates/dashboard.html

```diff
@@ -0,0 +1,185 @@
+
+{% block extra_scripts %}
+<script>
+  var ognSupplyHistory = {{supply_history | safe}};
+</script>
+{% endblock %}
+
+{% block content %}
+
+{% with theme_dark=True %}
+{% include 'navbar.html' %}
+{% endwith %}  
+
+<main class="dashboard-page">
+  <section class="title-section blue-section">
+    <div class="container">
+      <h1>{{gettext("OGN Dashboard")}}</h1>
+
+      <div class="row">
+
+        <div class="col-md-3 mx-md-0 px-md-0">
+          <div class="stats-widget">
+            <img src="/static/img/dashboard/price-icon.svg" />
+            <div class="stats-label">{{gettext("Price")}}</div>
+            <div class="stats-value">{{ogn_stats.ogn_price}}</div>
+          </div>
+        </div>
+
+        <div class="col-md-3 mx-md-0 px-md-0">
+          <div class="stats-widget">
+            <img src="/static/img/dashboard/circulating-supply-icon.svg" />
+            <div class="stats-label">{{gettext("Circulating Supply")}}</div>
+            <div class="stats-value">
+              {{ogn_stats.circulating_supply}}
+              <div class="ogn-label">OGN</div>
+            </div>
+          </div>
+        </div>
+
+        <div class="col-md-3 mx-md-0 px-md-0">
+          <div class="stats-widget">
+            <img src="/static/img/dashboard/market-cap-icon.svg" />
+            <div class="stats-label">{{gettext("Market Cap")}}</div>
+            <div class="stats-value">{{ogn_stats.market_cap}}</div>
+          </div>
+        </div>
+
+        <div class="col-md-3 mx-md-0 px-md-0">
+          <div class="stats-widget">
+            <img src="/static/img/dashboard/total-supply-icon.svg" />
+            <div class="stats-label">{{gettext("Total Supply")}}</div>
+            <div class="stats-value">
+              {{ogn_stats.total_supply}}
+              <div class="ogn-label">OGN</div>
+            </div>
+          </div>
+        </div>
+
+      </div>
+    </div>
+  </section>
+
+  <section class="schedule-section white-section">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-6">
+          <h3>{{gettext("Token Unlock Schedule")}}</h3>
+          <div class="desc">
+            {{gettext("This was the token release schedule we committed to in January for the next 4 years. In that blurb, we also highlight that we anticipate actual circulating supply to be significantly lower than modeled supply for X, Y, and Z reasons.")}}
+          </div>
+          <div class="chart-container">
+            <canvas id="unlock_schedule_chart" height="440"></canvas>
+          </div>
+        </div>
+        <div class="col-md-6">
+          <h3>{{gettext("Modeled vs. actual token release schedule")}}</h3>
+          <div class="desc">
+            {{gettext("Fusce condimentum ultrices est quis dapibus. Phasellus massa tortor, ultricies a imperdiet in, dapibus ac ligula. Duis suscipit odio sed sagittis finibus. Curabitur justo neque, scelerisque eu velit interdum, finibus varius purus.")}}
```

Comment:
Replace to english for the folks who are not Latin fluent :) Same comment down below.

Reply:
These are just placeholder texts. We were planning to get this to staging before looking into the content. Will make sure to update it before prod though.